src/CommentsGridPage.js

```diff
@@ -1,98 +1,36 @@
-import React, { useMemo } from 'react';
-import { groupBy } from 'lodash';
-import moment from 'moment';
+import React, { useEffect, useState } from 'react';
 import { Grid } from '@material-ui/core';
-import { useQuery } from 'react-query';
-import axios from 'axios';
 
 import { Comment } from './components/Comment';
 import { SortBySelector, SortEnum } from './components/SortBySelector';
 
-export const CommentsGridPage = () => {
-  // fetching comments
-  const comments = useQuery(
-    'comments',
-    async () => {
-      const response = await axios.request({
-        method: 'get',
-        url: 'https://www.reddit.com/r/wallstreetbets/comments/.json',
-        params: {
-          sort: 'new',
-          limit: 30, // not 100 for now because of bad performance of fetching user details individually
-        },
-      });
-
-      return response.data?.data?.children?.map((obj) => obj.data);
-    },
-    {
-      keepPreviousData: true, // so the data doesnt disappear
-      refetchInterval: 15 * 1000,
-    }
-  );
-
-  // fetching authors
-  const authors = useQuery(
-    ['authors', { comments: comments.data }],
-    async ({ queryKey: [, { comments }] }) => {
-      // TODO
-      // better to batch fetch the list but not sure how to do that ATM...
-      const authorsDetails = await Promise.all(
-        comments.map(async (comment) => {
-          const response = await axios.get(`https://www.reddit.com/user/${comment.author}/about/.json`);
-          return response.data.data;
-        })
-      );
+import socketIOClient from 'socket.io-client';
+const ENDPOINT = 'http://127.0.0.1:5000/';
```

Comment:
before we push this to master we need to configure it to work with heroku so the frontend team does not have to run the backend as well.

Reply:
Well it only works with the local version that I hacked up a bit. I'm working on both simultaneously to try to get a good interface between the two

Reply:
ok that makes sense imo I wouldn't push to master until the proxy is set up and the backend changes you're working on are pushed.